tools/tophat2/tophat_macros.xml

```diff
@@ -36,14 +36,20 @@
       </param>
       <when value="Yes">
         <conditional name="gene_model_ann">
-          <param name="use_annotations" type="select" label="Use Gene Annotation Model">
+          <param name="use_annotations" type="select" label="Use Gene Annotation Model" help="-G/--GTF; TopHat with a set of gene model annotations and/or known transcripts, as a GTF 2.2 or GFF3 formatted file. If this option is provided, TopHat will first extract the transcript sequences and use Bowtie to align reads to this virtual transcriptome first. Only the reads that do not fully map to the transcriptome will then be mapped on the genome. The reads that did map on the transcriptome will be converted to genomic mappings (spliced as needed) and merged with the novel mappings and junctions in the final tophat output. Please note that the values in the first column of the provided GTF/GFF file (column which indicates the chromosome or contig on which the feature is located), must match the name of the reference sequence in the Bowtie index you are using with TopHat.">
             <option value="No">No</option>
-            <option value="Yes">Yes</option>
+            <option value="Use a built-in gene annotation">Use a built-in gene annotation</option>
+            <option value="Use a gene annotation from history">Use a gene annotation from history</option>
```

Comment:
Having option values with spaces and this long is probably not good practice, I'd use something like:
```
            <option value="indexed">Use a built-in gene annotation</option>
            <option value="history">Use a gene annotation from history</option>
```